.github/workflows/ci.yml

```diff
@@ -7,22 +7,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        project: ['.', 'glfw', 'freetype']
+        project: [".", "glfw", "freetype"]
     # We want to run on external PRs, but not on our own internal PRs as they'll be run by the push
     # to the branch.
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     env:
-      DISPLAY: ':99.0'
+      DISPLAY: ":99.0"
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Zig
         run: |
-          sudo apt install xz-utils
+          sudo apt install xz-utils libasound2-dev
```

Comment:
Either this should be just `libasound2` (not `-dev`), or whatever header dependency we have in here should be distributed via https://github.com/hexops/sdk-linux-x86_64 and https://github.com/hexops/sdk-linux-aarch64 instead.

Reply:
(should address this before merging)

Reply:
i'd tried without `-dev` but it didn't work. i do the same in my machine (linux mint)

Reply:
What part about it didn't work? This would indicate cross-compilation won't work

Reply:
missing headers/static libs

Reply:
https://github.com/hexops/mach/issues/431